### ISSUE TITLE
No Fullscreen launcher option

### ISSUE THREAD

**Issue by shinra358** [xan105/FFX-X2_LAUNCHER#17]
This doesnt have the fullscreen launcher option like the old version. Dont see the custom image like the old version either.

**Comment by xan105** [comment]
I'm sorry but the older version didn't have the aforementioned features.

Regarding fullscreen, as I have mentioned in https://github.com/xan105/FFX-X2_LAUNCHER/issues/1#issuecomment-1492880169. I never implemented fullscreen because the launcher UI (excluding settings) is based on the original launcher using overlay images which are tricky to scale and maintain correct positioning and aspect ratio across different resolutions.

**Comment by shinra358** [comment]
1.06 had it. im using it now and its fullscreen. i was the one given u
feature suggestions in the steam gorums many years ago and it does in fact
have a custom image feature also. the fullscreen in the old version worked
just fine 🤔

**Comment by shinra358** [comment]
Proof: 
![20241103_033314](https://github.com/user-attachments/assets/6d0710f0-0e73-4e40-b876-662fd7b908ec)
![20241103_033351](https://github.com/user-attachments/assets/b88baf2b-14fe-4e58-98d6-43ac6eefaad4)

As you can see, in pic 1, launcher bg custom. And in image 2, automatic full screen with a custom image i made. You made it while i was on win7. Now these images are on win10. And ive only tested 3 resolutions: 1080p, 720p, and 4k and no scaling issues here on 1.06.

Only problem i have w 1.06 is when u exit the game, theres 2 instances of the launcher.

**Comment by xan105** [comment]
Yes I do remember your suggestions from back then. But, again and I say this respectfully, It doesn't look like my current and older launcher. Maybe it's from another project ? I honestly don't know.

**Comment by shinra358** [comment]
hmm, is lulech23 your alt alias or is that a different person? That's what it says in the additional credits. Somebody changed the timeline xD
